pkg/event-manager/subscriptions/http_handlers.go

```diff
@@ -63,7 +62,7 @@ func (h *Handlers) addSubscription(params subscriptionsapi.AddSubscriptionParams
 	defer sp.Finish()
 
 	if err := params.Body.Validate(strfmt.Default); err != nil {
-		return eventsapi.NewEmitEventBadRequest().WithPayload(&models.Error{
```

Comment:
Looks like this was here by mistake

Reply:
Good catch! It would be great if we could unify our models/error responses... maybe some day :)